lib/event_framework/event.rb

```diff
@@ -39,14 +39,15 @@ class SystemMetadata < BaseMetadata
       unattributed: UnattributedMetadata,
       system: SystemMetadata,
     }
+    MetadataSum = Metadata | UnattributedMetadata | SystemMetadata
```

Comment:
Naming-wise, having to put `Sum` on the end feels a tad awkward to me... I wonder if `AnyMetadata` might feel more natural?

Reply:
Force pushed, that's a much nicer name.